ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -736,4 +736,10 @@ FAN_SS_MSM:
   description: "Basic fan with speed monitoring (used in jet-fans for ventilation purpose)"
   implements:
   - FAN
-  - MSM
\ No newline at end of file
+  - MSM
+
+FAN_DSPZTC:
```

Comment:
given the abstract type you put together, this seems to me like it is easily covered by the existing type [FAN_SS_DSP](https://github.com/google/digitalbuildings/blob/afcd129422941b8c4d50bcf8e268b251b1c700fb/ontology/yaml/resources/HVAC/entity_types/FAN.yaml#L67). it would give you all the same fields, assuming the cooling and heating setpoints are analagous to your zone temp limits.

Reply:
Yes checked with people more knowledgeable than myself and this would work thank you.